src/test/regress/regress_gp.c

```diff
@@ -642,7 +646,12 @@ resGroupPalloc(PG_FUNCTION_ARGS)
 		PG_RETURN_INT32(0);
 
 	ResGroupGetMemInfo(&memLimit, &slotQuota, &sharedQuota);
-	size = ceilf(memLimit * ratio);
+	if (!startupConsidered) {
+		size = ceilf(memLimit * ratio) - 12;
```

Comment:
If 12 chunks are always required for the startup memory, maybe it should be factored out as a macro or a constant?

Reply:
No, it's just a mistake, there should've been a function that retrieves the actual number of startup chunks, but it happened to be 12 on my configuration. I fixed this